src/imagetransforms/CloudflareImagesTransformer.php

```diff
@@ -51,6 +51,21 @@ public function generateFlexibleVariant(ImageTransform|CloudflareImagesTransform
         return '/' . implode(',', $transform);
     }
 
+    public function positionToGravity(ImageTransform $imageTransform)
+    {
+        $positions_map = [
+            'left' => 0,
+            'right' => 1,
+            'top' => 0,
+            'bottom' => 1,
+            'center' => 0.5
+        ];
+        $positions = explode('-', $imageTransform->position);
+        $x = $positions[0] ?? 'center';
+        $y = $positions[1] ?? 'center';
```

Comment:
php requires you to check for undefined keys in arrays. It will crash if the exploded string does not contains an array.

We could add if (count($positions) == 2) and return an empty string or check for individual keys

Reply:
It doesn't seem to crash, but it did crash of I had a wrong key form the map, so I adjusted the code.

You can test it here: 

https://php-play.dev/?c=DwfgDgFmBQ0MYBsCGBnFACAkgWyQcwFMAVAJyQDsUAzAexO3QG9p1X0wBXAIwQEs50AEjA0UvAC68a5ANzQAvrESoMRAinFMWbTj37oqHcnEnT0hcgTLiCAMQQEAHrx4EAakhK8K4gBQ58YjJKWnohXlxCUgpqOmwASnRtVmY2NKFxYNiwgF50AG0AXTl0tl4qdF9BCMDokLiAWgA+ETFTckTU0vTBTJjQ7CL0PIAiPDIANwkATxzGXoheFGbWiSlyIhoAcUmZqpqorIH4+RGS7sVu9HLKgmwwcWmqvvr6eM7kq-QSAnEOEnI6AA5EDzqVLt0fn8AcCAPRA9AAOmu9wQNAAJgRfECADS4jJHOLxOSfRSfXR8ASGYztdiiNbSTY7JBTR7+SJBfpxcIcurZBKfLqlYT09ooAD6uDAwwKnyuQIcVHECJyTXQAAYcXLukCvHgIMrhmqAIxar5sIHiGhgFVqzXa0pArg0cRW7C29Cmh3pIFwAjkGwkD3qxEAVm9xW9Ira6wweScYDRmOxDXx1V5hPoK1F62JUccMujDMo+XVhXQIBAwL9AasoKj00LqzF+WN5cr1f9gfrV0EcBodHRAA0mznpBKpflBI521WQ6GwT1+4OAJqjmPjyVIMBT6azjVhxdpKH-QERRMYrFAxz4qfLkjDnFCe-oleFPNpRRk3rqTR5SwAO7oGoGi+HmvSZgw-4EEBASHFy9BgSSEEIdg2YboCeS+l2VgNDW3bIT8KAcAgf4ZL+zQWFYSA2PYTguA4HheD4zyQXm0AEHAEA0EIREkeIMhAA&v=8.3&f=html